presto-main/src/main/java/com/facebook/presto/execution/QueryPreparer.java

```diff
@@ -59,7 +59,13 @@ public PreparedQuery prepareQuery(Session session, String query)
     public PreparedQuery prepareQuery(Session session, Statement wrappedStatement)
             throws ParsingException, PrestoException, SemanticException
     {
-        Statement statement = unwrapExecuteStatement(wrappedStatement, sqlParser, session);
+        Statement statement = wrappedStatement;
+        String prepareSql = null;
+        if (wrappedStatement instanceof Execute) {
```

Comment:
Change this to `statement` so it matches the cast on the next line (even though they're the same it looks better)